SU-SWS/stanford_media
# D8CORE-1325 Make dropzone error more visible

# READY FOR REVIEW

# Summary
- Make dropzone error always visible.

# Need Review By (Date)
- :man_shrugging: 

# Urgency
- low

# Steps to Test
1. Checkout this branch
1. go to any dropzone area and attempt to upload an incorrect file. like a php file or a file that is over 5MB
1. validate you see an error even when you aren't hovering over the item.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
